README.md

```diff
@@ -0,0 +1,69 @@
+# React Auto Translate
+> Simple React components to automatically translate strings with Google Translate API
```

Comment:
I'm thinking this should be "components that" or "component to".... I think our tensing is a touch off.